platform/utm_mgrs_utils.cpp

```diff
@@ -0,0 +1,567 @@
+#include "platform/utm_mgrs_utils.hpp"
+
+#include "base/string_utils.hpp"
+
+#include <string>
+#include <iostream>
+#include <cmath>
+
+namespace utm_mgrs_utils
+{
+using namespace std;
+
+typedef struct UTMPoint_Value
+{
+  double easting;
+  double northing;
+  int zone_number;
+  char zone_letter;
+
+} UTMPoint;
+
+UTMPoint latlon_to_utm(double lat, double lon);
```

Comment:
Are these definitions really necessary? It will be harder to edit their signature in two places when necessary.

Reply:
Refactored functions. Removed forward declarations. Reordered functions.